app/views/partials/share.html

```diff
@@ -0,0 +1,12 @@
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:site" content="@openspending">
+<meta name="twitter:title" content="{{ shareMetadata.title | escape }}">
+<meta name="twitter:description" content="{{ shareMetadata.description | truncate(197, true) | escape }}">
+<meta name="twitter:image" content="http://staging.openspending.org/sealer/store?width=500&height=400&url={{ shareMetadata.url | urlencode | escape }}">
+
+<meta property="fb:app_id" content="966242223397117">
```

Comment:
Default app id (from FB docs). Also I've used recommended sizes for images (also from docs).